pkg/local_object_storage/metabase/put.go

```diff
@@ -112,15 +112,18 @@ func (db *DB) put(
 		return nil
 	}
 
-	if par := obj.Parent(); par != nil && !isParent { // limit depth by two
-		parentSI, err := splitInfoFromObject(obj)
-		if err != nil {
-			return err
-		}
+	par := obj.Parent()
```

Comment:
just to shorten `if` statement or u forgot to use `par` below?

Reply:
returned back, not sure why i needed it